CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Warn in case of isobaric counter diffusion
```

Comment:
Honestly I think that @willemferguson and your work does deserve a little bit more detailed description in the future release notes!
Maybe also add a comment about the rule of 1/5?

Reply:
I will insert suitable text in the user manual.